src/app/home/home.component.ts

```diff
@@ -30,7 +30,7 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
   current_flag = 'en';
   current_lang = 'English';
   sidenavState = 'closed';
-  notifications = [];
+  notifications = { count_unread: 0 };

```

Comment:
This is adding to the problem, as you later assign an array to this.notifications.  It should just be initialized as an array.